lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -892,8 +893,10 @@ namespace Js
 
         void SetCrossSiteForSharedFunctionType(JavascriptFunction * function);
 
+        bool IsPRNGSeeded() { return isPRNGSeeded; }
```

Comment:
If the scope of const is clear, it is okay to add. Personally I prefer it but if it causes any compiler breaks, it's not worth propagating the const keyword to fix them. In that case just revert.